### PR TITLE
Plat 658 use new gocli

### PR DESCRIPTION
**Pull request by bradsawadye**

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -7,3 +7,9 @@ CLUSTERED_MODE=false
 DEBUG=0
 BASHLOG_FILE=0
 BASHLOG_FILE_PATH=platform.log
+
+# Disables Fhir validation agains IG
+DISABLE_VALIDATION=true
+
+# Name of folder containing the Fhir IG importer scripts
+FHIR_CONFIG_IMPORTER_FOLDER_NAME=hapi-fhir
```
Comment (nour-borgi) [inline .env.local:15]: Same comment with Michael about defining the env vars only for override

File: .env.prod [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+DEBUG=0
+BASHLOG_FILE=0
+BASHLOG_FILE_PATH=platform.log
+
+# Disables Fhir validation agains IG
+DISABLE_VALIDATION=true
+
+# Name of folder containing the Fhir IG importer scripts
+FHIR_CONFIG_IMPORTER_FOLDER_NAME=hapi-fhir
```
Comment (nour-borgi) [inline .env.prod:15]: same here

File: .env.local (hunk 1 above)
Comment (bradsawadye) [reply .env.local]: addressed

File: .env.prod (hunk 2 above)
Comment (bradsawadye) [reply .env.prod]: addressed

File: package-metadata.json [hunk 3]
```diff
@@ -20,6 +20,8 @@
     "SUPERSET_SSL": "false",
 
     "FHIR_IG_URL": "https://openhie.github.io/cares-covid-ig",
```
Comment (Mark-Labuschagne) [inline package-metadata.json:22]: interoperability-layer-openhim should be in the dependencies

Comment (Mark-Labuschagne) [inline package-metadata.json:22]: interoperability-layer-openhim should be in the dependencies

**Review by Mark-Labuschagne — CHANGES_REQUESTED** [review]

File: package-metadata.json (hunk 3 above)
Comment (bradsawadye) [reply package-metadata.json:22]: Thanks, I agree with you but this is not related to this pr. You would also need to add a config importer for creating the openhim channel that will be used. That work should be done under a different ticket

File: config.yml [hunk 4]
```diff
@@ -1,25 +1,29 @@
----
-image: jembi/platform
-defaultTargetLauncher: swarm
-disableKubernetes: true
-disableIG: true
-disableCustomTargetSelection: true
+projectName: Cares-on-platform
+image: jembi/platform:latest
```
Comment (michaelloosen) [inline config.yml:2]: Shouldn't we pin this so that we don't have breaking changes in platform updates affect this package?

File: .gitignore [hunk 5]
```diff
@@ -74,6 +74,6 @@ typings/
 /docs/.cache-loader
 
 # Built Go Binaries
-platform-linux
-platform.exe
-platform-macos
+instant-linux
```
Comment (michaelloosen) [inline .gitignore:77]:
```suggestion
platform-linux
platform.exe
platform-macos
instant-linux
```

**Review by michaelloosen — CHANGES_REQUESTED** [review]

File: config.yml (hunk 4 above)
Comment (michaelloosen) [reply config.yml:2]:
```suggestion
image: jembi/platform:1.5.0
```

**Review by michaelloosen — APPROVED** [review]

**Review by nour-borgi — APPROVED** [review]

**Pull request merged**